nip47/event_handler.go

```diff
@@ -45,6 +45,14 @@ func (svc *nip47Service) HandleEvent(ctx context.Context, relay nostrmodels.Rela
 		}).Error("invalid event signature")
 		return
 	}
+	pTag := event.Tags.GetFirst([]string{"p"})
+	if pTag == nil {
```

Comment:
Why was this added? we can get it from the app, right?

the spec says `SHOULD` but here it is implemented as `MUST`

Reply:
My mistake, thanks!